common/src/cellular_3gpp_urc_handler.c

```diff
@@ -542,11 +542,11 @@ CellularPktStatus_t _Cellular_ParseRegStatus( CellularContext_t * pContext,
     uint8_t i = 0;
     char * pRegStr = NULL, * pToken = NULL;
     cellularAtData_t * pLibAtData = NULL;
-    CellularServiceStatus_t serviceStatus = { 0 };
+    CellularServiceStatus_t serviceStatus;
```

Comment:
Suggest we move this variable to _regStatusGenerateEvent function since it is only used in this function.

Reply:
That make sense, I will make this change.